extension-requests/style.css

```diff
@@ -23,6 +30,204 @@
   box-sizing: border-box;
 }
 
+.card-more-details {
+  background-color: grey;
+}
+
+.extension-card {
+  border-radius: 10px;
+  text-align: left;
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+  margin-bottom: 1.5rem;
+  justify-content: space-between;
+  border: 1px solid var(--medium-gray);
+  position: relative;
+  max-width: 48rem;
+  width: 100%;
+  gap: 1rem;
+}
+
+.accordion {
+  background-color: transparent;
+  cursor: pointer;
+  width: 100%;
+  border: none;
+  text-align: left;
+  outline: none;
+  font-size: 1rem;
+  transition: 0.8s;
+  display: flex;
+  justify-content: center;
+}
+
+.details-container {
+  display: flex;
+  gap: 4px;
+  align-items: center;
+}
+
+.accordion img {
+  transition: transform 500ms ease, scale 100ms ease;
+}
+.accordion:hover img {
+  transform: scale(1.25);
+}
+.accordion.active img {
+  transform: rotate(180deg);
+}
+
+.assignee-container {
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+}
+.assignee-image {
+  width: 1.875rem;
+  height: 1.875rem;
+  color: transparent;
+  border-radius: 50%;
+}
+.assignee-name {
+  font-size: 1.3rem;
+  font-weight: 500;
+  border-radius: 5px 5px 5px 5px;
```

Comment:
- can be replaced to `border-radius: 5px` ?

Reply:
done.